lib/mastodon/version.rb

```diff
@@ -9,7 +9,7 @@ def major
     end
 
     def minor
-      4
+      5
```

Comment:
This is actually the release version. The next release will definitely be 4.4, so this is wrong.

You probably meant to change the API version. This is an internal version number our official apps make use of to determine which features the API supports. This is defined a couple of lines below in the `#api_versions` method.

Reply:
Oops, sorry I overlooked that!